core/templates/dev/head/pages/dashboard/Dashboard.js

```diff
@@ -215,3 +209,23 @@ oppia.controller('Dashboard', [
     );
   }
 ]);
+
+oppia.directive('explorationTitle', function($compile) {
+  return {
+    restrict: 'E',
+    scope: {
+      exploration: '='
```

Comment:
Why pass in the whole `exploration` object when only exploration.status is needed ?